client/src/components/organisms/TaskSlider/index.tsx

```diff
@@ -130,19 +133,11 @@ const TaskSlider: FC<Props> = (props): JSX.Element => {
     useHandleUpdateTaskAssignee(parseInt(task_id as string), getMember.id)
     handleUpdateAssigneeToggle()
   }
-  const onSearchChange = (e: any): void => {
-    const value = e.target.value
-    if (value.length === 0) {
-      filterMembersName(parseInt(id as string), value)
-    }
-  }
-  const onSearch = (e: any): void => {
-    const value = e.target.value
-    const isEnter = e.key === 'Enter' || e.keyCode === 13
 
-    if (!isEnter) return
-    filterMembersName(parseInt(id as string), value)
-  }
+  useEffect(() => {
+    filterMembersName(parseInt(id as string), search)
+  }, [debouncedSearch])
+
   const handleSetName = (e: any) => {
     const keyCode = e.which || e.keyCode
```

Comment:
Replace any with it's appropriate type.

Reply:
this is an event, which you said it is any at the moment